crates/core/tedge_actors/src/converter.rs

```diff
@@ -1,3 +1,59 @@
+//! Actors that convert each input message into a sequence of output messages
```

Comment:
I didn't know the converter, now I know it! Looks so nice!!!

Reply:
Yes, there something to improve here.